### ISSUE TITLE
Are there any other unblockers you know of?

### ISSUE THREAD

**Issue by neelansh2489** [nfriedly/node-unblocker#117]
Vikingpress got blocked in our school so please please help!

**Comment by nfriedly** [comment]
Not that I know of, but you can run this from pretty much any computer.

**Comment by easella** [comment]
you can use these:
pydodge:https://maple-butter.herokuapp.com/course/https://youtube.com(youtube is fully functional. works with aquapark io and yohoho io)
womginx: https://extremely-fast-dog.herokuapp.com/lesson/https://google.com (works with hulu, discord,and youtube video embeds!)(also supports login forms and cookies.)(does not work with google login.)
alloyproxy:http://partialvirtualredundancy-1.awdrgyjil1234.repl.co/ (works with many io games including zombsroyale and surviv)
paper.iofreshman.com/proxy/https://google.com ( works with krunt io, wormax io, and slither io)
powermouse: http://powermouseproxy.herokuapp.com/ (works with google and many io games)

**Comment by nfriedly** [comment]
Interesting. pydodge and use https://github.com/webrecorder/wombat on the front-end, so probably one of the associated projects on the back-end.

alloyproxy one uses https://github.com/titaniumnetwork-dev/alloy

The paper.iofreshman.com looks like it includes code that I wrote for this proxy, from the websockets branch before I finished that and got it merged in.

and, finally, powermouse uses https://github.com/vibedivide/powermouse

**Comment by easella** [comment]
yes. there is systemya.https://github.com/sysce/proxy

demo


| 
| 
|  | 
Sy​ste​mYA

Web access service
 |

 |

 |

demo2demo3demo4demo5


    On Tuesday, April 6, 2021, 09:02:42 PM EDT, Nathan Friedly ***@***.***> wrote:  
 
 


Interesting. pydodge and womginx use https://github.com/webrecorder/wombat on the front-end, so probably one of the associated projects on the back-end.

alloyproxy one uses https://github.com/titaniumnetwork-dev/alloy

The paper.iofreshman.com looks like it includes code that I wrote for this proxy, from the websockets branch before I finished that and got it merged in.

and, finally, powermouse uses https://github.com/vibedivide/powermouse

**Comment by easella** [comment]
oh yeah i forgot.there is slither.iofreshman.com there is wormax.iofreshman.com there is gartic.iofreshman.com 
there is stabfish.iofreshman.com 
there is evowars.iofreshman.comthere is surviv.iofreshman.com  



there is slither.iostudies.com there is wormax.iostudies.com there is paper.iostudies.com 
there is gartic.iostudies.com 
there is stabfish.iostudies.com 
there is evowars.iostudies.comthere is surviv.iostudies.com  



there is slither.ioclasses.com there is wormax.ioclasses.com there is paper.ioclasses.com 
there is gartic.ioclasses.com 
there is stabfish.ioclasses.com 
there is evowars.ioclasses.comthere is surviv.ioclasses.com  
    On Tuesday, April 6, 2021, 09:02:42 PM EDT, Nathan Friedly ***@***.***> wrote:  
 
 


Interesting. pydodge and womginx use https://github.com/webrecorder/wombat on the front-end, so probably one of the associated projects on the back-end.

alloyproxy one uses https://github.com/titaniumnetwork-dev/alloy

The paper.iofreshman.com looks like it includes code that I wrote for this proxy, from the websockets branch before I finished that and got it merged in.

and, finally, powermouse uses https://github.com/vibedivide/powermouse

**Comment by easella** [comment]
there is blocktanks.iofreshman.com
there is superhex.iofreshman.com
there is blocktanks.iostudies.com
there is superhex.iostudies.com
there is blocktanks.ioclasses.com
there is superhex.ioclasses.com

    On Tuesday, April 6, 2021, 09:02:42 PM EDT, Nathan Friedly ***@***.***> wrote:  
 
 


Interesting. pydodge and womginx use https://github.com/webrecorder/wombat on the front-end, so probably one of the associated projects on the back-end.

alloyproxy one uses https://github.com/titaniumnetwork-dev/alloy

The paper.iofreshman.com looks like it includes code that I wrote for this proxy, from the websockets branch before I finished that and got it merged in.

and, finally, powermouse uses https://github.com/vibedivide/powermouse
